lib/src/common/disposable.dart

```diff
@@ -74,6 +74,17 @@ class _ObservableTimer implements Timer {
   bool get isActive => _timer.isActive;
 }
 
+/// A class used as a marker for potential memory leaks.
+class LeakFlag {
+  final String description;
+
+  LeakFlag(this.description);
+
+  @override
+  String toString() =>
```

Comment:
It seems trivially correct to me and there's no reason it has to be anything in particular since we don't really use it anywhere.